lib/fastlane/plugin/ddg_apple_automation/actions/tag_release_action.rb

```diff
@@ -24,7 +24,6 @@ def self.setup_constants(platform)
 
       def self.run(params)
         platform = params[:platform] || Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
-        other_action.ensure_git_branch(branch: "^(release|hotfix)/#{platform}/.+$")
```

Comment:
We need to remove this check because now a commit is tagged – instead branch check happens in the calling workflow.